src/views/FAQ.vue

```diff
@@ -32,33 +32,29 @@
           <a
             href="https://aeternity.com/"
             target="_blank"
-          >aeternity blockchain</a>.
-          <br>Anyone, anywhere may use the platform.
-          <br>Other websites may use the platform.
-          <br>All have uncensored, equal ability to be Superhero.
-          <br>As P2P – the only ones who are not part of Superhero intermediaries.
-          <br>No 3
-          <sup>rd</sup> party custodians will control
-          content, or your ability to tip or receive tips.
-          <br>Superhero is you and you and you and you… Without any “them” in-between.
+          >aeternity blockchain</a>. <br>Anyone,
+          anywhere may use the platform. <br>Other websites may use the platform. <br>All have
+          uncensored, equal ability to be Superhero. <br>As P2P – the only ones who are not part
+          of Superhero intermediaries. <br>No 3 <sup>rd</sup> party custodians will control
+          content, or your ability to tip or receive tips. <br>Superhero is you and you and you
+          and you… Without any “them” in-between.
         </expandable-block>
         <expandable-block :title="'How does Superhero work?'">
           Superhero is built on aeternity blockchain.
-          <br>Because it is built on blockchain, actions are transparent.
-          <br>Because it is built on the blockchain, actions are published forever.
-          <br>Blockchain has promised a more free and decentralized future for society.
-          <br>Superhero used blockchain as its foundation to make that promise a reality.
-          <br>Superhero is an online community making a tangible difference in the real world.
+          <br>Because it is built on blockchain, actions are transparent. <br>Because it is
```

Comment:
the linter requires `<br>` to be the last on the line with a break after it (AFAIK)

Reply:
Tough not a problem for me per se

Reply:
> the linter requires `<br>` to be the last on the line with a break after it (AFAIK)

Strange, because I run it before every commit and it does not complain.

Reply:
I think it is a warning which fails on travis, but might not fail locally

Reply:
But if it passes then just don't change it. May be i'm mistaken